views/footer.twig

```diff
@@ -0,0 +1,17 @@
+<footer class="text-muted">
+    <div class="container">
+        <p class="float-right">
+            <a href="#">Back to top</a>
+        </p>
+        <p>ArcTouch - CodeChallenge. Applicant: Matheus Muller Borges</p>
+    </div>
+</footer>
+
+<!-- Bootstrap core JavaScript
+================================================== -->
+<!-- Placed at the end of the document so the pages load faster -->
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script>window.jQuery || document.write('<script src="/assets/js/vendor/jquery.min.js"><\/script>')</script>
```

Comment:
What is the advantage of the `document.write` here?

Reply:
The `document.write` is being used to load dynamically the local jQuery library in case the file from line 13 can't be loaded (e.g. CDN server is down).